src/main/webapp/appscript.js

```diff
@@ -13,27 +13,73 @@
 // limitations under the License.
 
 /**
-  * Fetches vendor IDs from VendorServlet and adds them to the
+  * Fetch vendor IDs from VendorServlet and add them to the
   * Customer ID dropdown.
   */
 async function populateCustomerList() {
+  // TODO: Implement functionality for hashmap parsing instead of simple array.
   const response = await fetch('/VendorServlet');
 
+  // VendorServlet returns a hashmap/JSON dictionary. 
+
   // Response is a json array that is parsed into json.
-  var customer_json = await response.json();
+  var customerJson = await response.json();
+
+  console.log(customerJson);
 
   // Loops over this array to make the options for the Customer ID dropdown.
   var html = '';
-  for (var i = 0; i < customer_json.length; i++) {
-    html += '<option value="' + customer_json[i] + '">'
-    + customer_json[i] + '</option>';
+  for (var i = 0; i < customerJson.length; i++) {
+    html += '<option value="' + customerJson[i] + '">'
+    + customerJson[i] + '</option>';
   }
 
   // Adds the options to the page, allowing them to be viewed. 
   document.getElementById('customer-ids').innerHTML = html;
 }
 
-// TODO: Fetch method for putting the specified config to the page.
+/**
+ * Fetch a json configuration as a string and formats it to be shown on
+ * the readfile page.
+ */
+async function addToPage() {
+  // TODO: Add query string support to sent to servlet.
+  var vendorID = document.getElementById('customer-ids');
+  
+  // Fetch the json configuration and format it to print on the page. 
+  fetch('/BillingConfig')
+    .then(response => response.json())
+    .then(data => document.getElementById('json-text').innerText = 
+    JSON.stringify(data, undefined, 4));
+    
+}
+
 // TODO: Method to populate edit form.
+async function populateEditForm() {
```

Comment:
Does it make sense to split off the javascript for the other pages into other JS files?

i.e. read_file.js, edit_file.js, etc...

Reply:
Still curious about this piece -- should these be in separate files? It may make life easier, especially with Charlie working in parallel on some of the front end stuff

Reply:
Please consider this and reply with your path forward.

Reply:
Yeah, I'll make separate files for these

Reply:
opened an issue for this #52